packages/controllers/src/controllers/SendController.ts

```diff
@@ -149,26 +147,20 @@ export const SendController = {
         sendTokenAmount: this.state.sendTokenAmount,
         decimals: this.state.token.quantity.decimals
       })
-    } else if (
-      this.state.receiverAddress &&
-      this.state.sendTokenAmount &&
-      this.state.gasPrice &&
-      this.state.token?.quantity.decimals
-    ) {
+    } else {
       EventsController.sendEvent({
         type: 'track',
         event: 'SEND_INITIATED',
         properties: {
           isSmartAccount: activeAccountType === W3mFrameRpcConstants.ACCOUNT_TYPES.SMART_ACCOUNT,
-          token: this.state.token?.symbol,
+          token: this.state.token.symbol || '',
```

Comment:
Is removing optional property access is safe in any condition? If somehow token is undefined or API return value without token, it might throw error

Reply:
we check for the token a couple of lines up ahead